src/util/misc/findScaleTo.ts

```diff
@@ -1,4 +1,4 @@
-interface IWithDimensions {
+export interface IWithDimensions {
```

Comment:
can we use `TSize` instead? Or are the comments important?

Reply:
this was a ts compiler issue, i just added export to get it passing.
Not sure if TSize allow for passing objects that have more than width and height, but i can try it